Master-OB-OpenAPI.json

```
@@ -17684,6 +17704,43 @@
     },
     "DateTimeItemType": {
       "description": "ISO-8601 formatted date/time string."
+    },
+    "JobStatusItemType": {
+      "description": "Represents the working status of a Job.",
+      "enums": {
+        "SalesQualification": {
+          "label": "Sales-Qualification",
+          "description": ""
+        },
+        "Working": {
+          "label": "Working",
+          "description": ""
+        },
+        "OnHold": {
+          "label": "On-Hold",
+          "description": ""
+        },
+        "Cancelled": {
+          "label": "Cancelled",
+          "description": ""
+        },
+        "Completed": {
+          "label": "Completed",
+          "description": ""
+        },
+        "SalesPostponed": {
+          "label": "Sales-Postponed\t",
```

delete tab character
```suggestion
          "label": "Sales-Postponed",
```